src/pages/_layout.svelte

```diff
@@ -79,7 +81,7 @@
     }
 
     main {
-      min-height: calc(100% - var(--height-nav));
+      min-height: 100%;
```

Comment:
This has the potential to break a lot on mobile (see our conversation on Slack), has this been thoroughly tested?